core/http/client/src/main/java/org/eclipse/rdf4j/http/client/RDF4JProtocolSession.java

```diff
@@ -204,7 +207,7 @@ private synchronized String getTransactionURL() {
 	 *-----------------*/
 
 	public TupleQueryResult getRepositoryList()
-			throws IOException, RepositoryException, UnauthorizedException, QueryInterruptedException {
+		throws IOException, RepositoryException, UnauthorizedException, QueryInterruptedException {
```

Comment:
It looks like your formatter is still applying the single-indent rule, rather than sticking to 2 indents for wrapped lines. Did I make a mistake when I merged the formatting changes, or is there something out of sync on your end?

Reply:
I’ll try to sync it up again.